saleor/core/schedules.py

```diff
@@ -0,0 +1,78 @@
+from collections import namedtuple
+from datetime import datetime
+
+import pytz
+from celery.schedules import BaseSchedule
+from celery.utils.time import maybe_timedelta, remaining
+from django.db.models import Q
+
+schedstate = namedtuple("schedstate", ("is_due", "next"))
+
+
+class sale_webhook_schedule(BaseSchedule):
```

Comment:
I used here lowercase with an underscore as all celery schedules are written this way. Also, the [PEP](https://peps.python.org/pep-0008/#class-names) is saying:

> The naming convention for functions may be used instead in cases where the interface is documented and used primarily as a callable.

Reply:
Should we add this info to the docstring?

Reply:
Added.